src/site/content/en/blog/web-payments-updates/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Updates to the Web Payments APIs
```

Comment:
I've already merged this change, but I consider "Web Payments" as a proper noun. Do you still suggest to lower case them?

Reply:
Ah, I didn't notice that the working group was referring to things like the "Web Payments ecosystem." I thought this was just a descriptor for a class of APIs. Since the working group is treating it like a proper noun, I agree that we should too. Sorry about that.